release-notes/unreleased/921-search-for-profession.yml

```
@@ -0,0 +1,6 @@
+issue_key: 921
+show_in_stores: true
+platforms:
+  - android
+  - ios
+de: Man kann nun nach Berufen suchen
```

I think we prefer "Nominalisierte" sentences
```suggestion
de: Eine Suche für Berufe wurde hinzugefügt.
```